index.html

```diff
@@ -210,22 +210,21 @@ <h3>New Technical Reports</h3>
     </dd>
   </dl>
 
-  <p>The Working Group MAY develop additional Recommendation-track and non-Recommendation-track Technical
-  Reports.</p>
+  <p>The Working Group MAY develop additional Recommendation-track and non-Recommendation-track specifications.</p>
```

Comment:
I suggest reverting this because not all non-Recommendation-track Technical Reports will be specifications.

Reply:
I've made a mistake here. I should only include rec-track here. Since this is a section for normative specs.

Reply:
updated here.